examples/default/copies.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": "https://github.com/Inria-Visages/blob/master/BIDS-prov/context.json",
+  "@id": "http://example.org/ds00000X",
+  "generatedAt": "2020-03-10T10:00:00",
+  "wasGeneratedBy": {
+      "@id": "INRIA",
+      "@type": "Project",
+      "startedAt": "2016-09-01T10:00:00",
+      "wasAssociatedWith": { "@id": "NIH",
+                              "@type": "Organization",
+                              "hadRole": "Funding"
+                            }
+    },
+  "@graph": [
+    {
+      "prov:Activity" : [
+        {
+          "@id" : "copy",
```

Comment:
"copy" should really be a label (as otherwise we have to create a term for it) and I would use a UUID as id, something like: 
```
"@id" : "653467354367537gjsfhdgfhjsf67345523efdghs",
"prov:label" : "copy",
```